spec/parallel_tests/rspec/runner_spec.rb

```diff
@@ -119,15 +119,40 @@ def call(*args)
       ParallelTests::RSpec::Runner.send(:find_tests, *args)
     end
 
-    it "finds turnip feature files" do
-      with_files(['a/test.feature']) do |root|
-        expect(call(["#{root}/"])).to eq(["#{root}/a/test.feature"])
+    context 'without --tag' do
```

Comment:
these tests don't really test the outcome, but how things pass around under the hood, so kind of easy to break and hard to understand
... we we instead test an outcome like that it runs the tests that have the given tag and not the others ?